mailpoet/lib/Automation/Integrations/MailPoet/Analytics/Controller/OverviewStatisticsController.php

```diff
@@ -0,0 +1,137 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\Automation\Integrations\MailPoet\Analytics\Controller;
+
+use MailPoet\Automation\Engine\Data\Automation;
+use MailPoet\Automation\Engine\Data\Step;
+use MailPoet\Automation\Integrations\MailPoet\Actions\SendEmailAction;
+use MailPoet\Automation\Integrations\MailPoet\Analytics\Entities\Query;
+use MailPoet\Entities\StatisticsClickEntity;
+use MailPoet\Entities\StatisticsOpenEntity;
+use MailPoet\Newsletter\NewslettersRepository;
+use MailPoet\Newsletter\Statistics\NewsletterStatisticsRepository;
+use MailPoet\Newsletter\Statistics\WooCommerceRevenue;
+use MailPoet\WooCommerce\Helper;
+
+class OverviewStatisticsController {
+
+    /** @var NewslettersRepository */
+    private $newslettersRepository;
+
+    /** @var NewsletterStatisticsRepository */
+    private $newsletterStatisticsRepository;
+
+    /** @var Helper */
+    private $wooCommerceHelper;
+
+  public function __construct(
```

Comment:
Something wrong with the indents again, above and below, in class properties and in function bodies. Does your IDE setting maybe not understand some of our indentation? I know I always say a sniff should watch this, but for some reason it doesn't.

Reply:
:face_exhaling: done